src/components/workspace-utils.js

```diff
@@ -12,7 +12,7 @@ import { MarkdownEditor, MarkdownViewer } from 'src/components/markdown'
 import Modal from 'src/components/Modal'
 import NewWorkspaceModal from 'src/components/NewWorkspaceModal'
 import TooltipTrigger from 'src/components/TooltipTrigger'
-import { Ajax } from 'src/libs/ajax'
+import { Ajax, useAjax } from 'src/libs/ajax'
```

Comment:
Is the import of `Ajax` still needed?

Reply:
Some functions in this file still use the imported Ajax directly instead of retrieving it from the context.

Reply:
I see… so there was only a single Ajax call from this file that the cloud environments page used.

Reply:
Yes, and in the spirit of keeping the scope of this change as small as possible, I went with using it only for that one call. Not an easy decision.